### DISCUSSION TITLE
What does "Import DDLC save data" mean and how do I know my data isn't imported already?

### DISCUSSION THREAD

**Discussion by emcchiaki** [Monika-After-Story/MonikaModDev#8332] [QA - tech support + general] [answered]
What does "Import DDLC save data" mean and how do I know my data isn't imported already?

I have played several DDLC routes by re-installing the game and I only had one Monika After Story ever. Monika remembers I deleted her, because there's a topic to ask if she's still mad at me deleting her. The thing is, I'm wondering if once I click the option to check it, it will somehow change my relationship with Monika? That's why I'd like to know if it will change anything and if so, what exactly, and what kind of data will be imported.

I downloaded my Doki Doki game from Steam and exported the files to my desktop's screen. Before I was downloading from the game's website ddlc.moe and itchio. (Maybe this info will help somehow)

**Comment by dreamscached** [comment, marked as answer]
It won't have much effect on your relationship, but will give Monika of memories she had in the base game, such as which girl did you choose to go on festival with.

**Comment by ibm2431** [comment]
One of the topic's conditions for appearing is if `persistent.monika_kill` is set. This is only set to true when importing save data.
